conjur-api/Variable.cs

```diff
@@ -47,17 +48,37 @@ public string GetValue()
         /// </summary>
         /// <param name="val">Secret value.</param>
         public void AddSecret(string val)
+        {
+            byte[] data = Encoding.UTF8.GetBytes(val);
+            AddSecret(data);
+        }
+
+        /// <summary>
+        /// Set a secret (value) to this variable.
+        /// The value is being cleared after used!!!
```

Comment:
It's actually our documentation so we need to elaborate why we clean the buffer after use. Please see [remarks](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/remarks) and [para](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/para) tags.

Reply:
Add comment